generator/sources/csharp/KalturaClientTester/KalturaUploadThread.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Threading;
+
+namespace Kaltura
+{
+    class KalturaUploadThread

```

Comment:
Please place this class as a facility under KalturaClient instead of under KalturaClientTester.